adapter/validator/src/main/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidator.java

```diff
@@ -95,39 +99,117 @@ public GtfsTypeValidator(@NotNull GtfsSpecificationProto.CsvSpecProto fileSchema
                     case INPUT_TYPE_UNSPECIFIED: // Text is default and does not require validation
                     case TEXT:
                     case COLOR:
-                    case DATE: //no special validation
+                    case DATE: // no special validation
                         break;
+                    case INTEGER:{
+                        if (!integerValidator.isInRange(
+                                (Integer) value,
+                                Integer.MIN_VALUE,
+                                Integer.MAX_VALUE)) {
+
+                            toReturn.add(new IntegerFieldValueOutOfRangeNotice(
+                                    toValidate.getRawFileInfo().getFilename(),
+                                    columnSpecProto.getName(),
+                                    toValidate.getEntityId(),
+                                    Integer.MIN_VALUE,
+                                    Integer.MAX_VALUE,
+                                    (Integer) value)
+                            );
+                        }
+                        break;
+                    }
                     case FLOAT: {
```

Comment:
Not defined in https://gtfs.mobilitydata.org/specifications/static-gtfs#h.hc443y62gb8c

Reply:
As discussed, will open a conversation with our spec team. Meanwhile, we'll keep this type.